src/pages/entry.js

```diff
@@ -1,19 +1,53 @@
-import React from 'react'
+/** @jsx jsx */
+import { jsx } from '@emotion/core'
+import React, { Fragment } from 'react'
 import { withRouter } from 'react-router-dom'
+import styles from './entry.styles'
 import Anchor from 'components/Anchor'
+import Box from 'components/Box'
+import LargeHeader from 'components/Header/LargeHeader'
+import ListLink from 'components/ListLink'
+import Text from 'components/Text'
+import { TEXT_TYPE } from 'components/Text/constants'
 
 class EntryPortal extends React.Component {
   constructor(props) {
-    super(props);
+    super(props)
   }
 
   render() {
     return (
-      <div>
-        <Anchor href="/request">Need</Anchor>
-      </div>
+      <Fragment>
+        <LargeHeader />
+        <Box css={styles.box}>
+          <ListLink
+            href={'/request'}
```

Comment:
```suggestion
            href="/request"
```

I'd keep string literals as-is, it's a tidbit easier to read.